pkg/frontend/admin_ovsflows_get.go

```diff
@@ -0,0 +1,173 @@
+	pod, err := createOVSFlowPod(ctx, node.Name, config, log)
+	if err != nil {
+		log.Error(err)
+		return nil, err
+	}
+
+	//  The below loop waits of the phase of the created pod to be Succeeded, if not it times out after 60 seconds and returns error.
+	for i := 0; pod.Status.Phase != corev1.PodSucceeded; i++ {
+		if i == 60 {
+			err = api.NewCloudError(http.StatusNotFound, api.CloudErrorCodeInternalServerError, "", "")
+			return nil, err
+		}
+		pod, err = config.CoreV1().Pods("default").Get(ctx, pod.Name, metav1.GetOptions{})
+		if err != nil {
+			log.Error(err)
+			return nil, err
+		}
+		time.Sleep(time.Second)
+	}
+
+	// The OVS flows would be present in the logs of the created pod.
+	b, err := getPodLogs(ctx, log, config, pod)
+	if err != nil {
+		return nil, err
+	}
+
+	// Clearing the pod after extracting its logs.
+	err = config.CoreV1().Pods("default").Delete(ctx, pod.Name, metav1.DeleteOptions{})
+	if err != nil {
+		log.Error(err)
+		return nil, err
+	}
+
+	return b, nil
+}
+
+// This Function extract logs from the passed pod and return in the form of a slice bytes.
+func getPodLogs(ctx context.Context, log *logrus.Entry, config *kubernetes.Clientset, pod *corev1.Pod) ([]byte, error) {
+	opts := corev1.PodLogOptions{Container: "container-00"}
+	r := config.CoreV1().Pods("default").GetLogs(pod.Name, &opts)
+	result, err := r.Do(ctx).Raw()
+	if err != nil {
+		log.Error(err)
+		return nil, err
+	}
+	return result, err
+}
+
+//
+func createOVSFlowPod(ctx context.Context, node string, config *kubernetes.Clientset, log *logrus.Entry) (*corev1.Pod, error) {
+	var hostpathType corev1.HostPathType = "Directory"
+	hostPath := corev1.VolumeSource{HostPath: &corev1.HostPathVolumeSource{"/", &hostpathType}}
+	volumes := []corev1.Volume{{"host", hostPath}}
+	privileged := true
+	var runAsUser int64 = 0
+
+	pod := &corev1.Pod{
+		metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		metav1.ObjectMeta{
+			Name:      "aro-ovsflows",
+			Namespace: "default",
+			//Annotations: annotations,
+		},
+		corev1.PodSpec{
+			Volumes: volumes,
+			Containers: []corev1.Container{{
+				Name:    "container-00",
+				Image:   "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:103505c93bf45c4a29301f282f1ff046e35b63bceaf4df1cca2e631039289da2",
+				Command: []string{"chroot", "/host", "/bin/bash", "-c", "ovs-ofctl -O OpenFlow13 dump-flows br0"},
+				VolumeMounts: []corev1.VolumeMount{{
+					Name:      "host",
+					MountPath: "/host",
+				}},
+				SecurityContext: &corev1.SecurityContext{
+					Privileged: &privileged,
+					RunAsUser:  &runAsUser,
+				},
+			}},
+			RestartPolicy: "Never",
+			NodeName:      node,
+			HostNetwork:   true,
+			HostPID:       true,
+		},
+		corev1.PodStatus{},
+	}
+
+	pod, err := config.CoreV1().Pods("default").Create(ctx, pod, metav1.CreateOptions{})
+	if err != nil {
+		log.Error(err)
+		return nil, err
+	}
```

Comment:
What do we do if the pod already exists? This might happen if we didn't manage to clean up properly from the previous run

Reply:
Added a logic to that .. will update the PR shortly with all the update. If the pod already exists, the name of the pod to be created will be change to podName-1, if this also exists the number will keep on increasing.